### PR TITLE
fix: the way to get email from token

### LINKED ISSUE #272: fix GetUserService
## Bug Description

It got auth token instead of a raw header like `Bearer authtoken`

### PR DESCRIPTION
**Pull request by taniguchiiqqq**
## Overviews of implementation
fixed that GetUserService couldn't get emailAddress from authToken

## Review points
Do you think that it's allowed to use `any` type

### CODE REVIEW

File: backend/main/src/Services/GetUserIdService/GetUserService.ts [hunk 1]
```diff
@@ -17,11 +17,12 @@ export class GetUserService implements IGetUserService {
 		}
 		// get token from bearer token
 		const authToken = authHeader.split(" ")[1];
-		const decodedToken = jwt.decode(authToken);
+		// biome-ignore lint/suspicious/noExplicitAny: allow any. This is specified by the Cognito.
+		const decodedToken: any = jwt.decode(authToken);
 		if (!decodedToken)
 			return Err(new InvalidTokenError("The token is invalid."));
```
Comment (hitohata) [inline backend/main/src/Services/GetUserIdService/GetUserService.ts:23]: Would you be willing to add the type-checking? The reason that you want to assign any to the variable might be an IDE error. That error comes from the decodedToken, which can be a string or a JwtPayload. By adding this check, an IDE can infer the variable is the JwtPayload.

```suggestion
		if (typeof decodedToken === "string")
			return Err(new InvalidTokenError("The token is invalid."));

```

Comment (taniguchiiqqq) [reply backend/main/src/Services/GetUserIdService/GetUserService.ts:23]: Thank you for pointing it out! I updated it

Comment (hitohata) [reply backend/main/src/Services/GetUserIdService/GetUserService.ts:23]: Oh, I mean that by adding this check, we can remove the 'any' annotation and the biome-ignore.

Comment (taniguchiiqqq) [reply backend/main/src/Services/GetUserIdService/GetUserService.ts:23]: Omg, I didn't notice the solution. I got it. I didn't realize JwtPayload type has `[key: string]`. I just thought the type didn't have an email property.  Thank you so much! I fixed it.

**Review by hitohata — APPROVED** [review]
LGTM

**Pull request merged**